component/authentication-endpoint/src/main/webapp/x509CertificateError.jsp

```diff
@@ -57,6 +57,21 @@
                     errorMessage = resourceBundle.getString("not.valid.certificate");
                 } else if (errorCode.equalsIgnoreCase("17003")) {
                     errorMessage = resourceBundle.getString("fail.validation.certificate");
+                } else if (errorCode.equalsIgnoreCase("17004")) {
+                    errorMessage = resourceBundle.getString(
+                            "x509certificateauthenticator.alternativenames.regex.multiplematches.error.message");
+                } else if (errorCode.equalsIgnoreCase("17005")) {
```

Comment:
could have used X509CErtificateConstants.X509_CERTIFICATE_ALTERNATIVE_NAMES_REGEX_NO_MATCHES_ERROR_CODE

Reply:
There was no imports from the authenticator to authenticator-endpoint. That might be to deploy the war file separately. hence It is risky to refer constants from a single location